src/client/src/client.rs

```diff
@@ -28,21 +28,21 @@ use tonic::transport::Channel;
 use crate::load_balance::{LoadBalance, Loadbalancer};
 use crate::{error, Result};
 
-pub(crate) struct DatabaseClient {
-    pub(crate) inner: GreptimeDatabaseClient<Channel>,
+pub struct DatabaseClient {
```

Comment:
Can we move this struct along with `Database`, too?

Reply:
I tried before but 
https://github.com/GreptimeTeam/greptimedb/blob/ed8b13689e122603d5127d000744fb756e1fddc2/src/client/src/client.rs#L167-L174
needs to expose following two method
https://github.com/GreptimeTeam/greptimedb/blob/ed8b13689e122603d5127d000744fb756e1fddc2/src/client/src/client.rs#L141-L149
so I keep it unchanged, if it is OK to do so, I can make changes

Reply:
I think it's ok to pub `max_grpc_recv_message_size`.